snare/cloner.py

```diff
@@ -182,10 +183,12 @@ async def get_body(self, driver):
             self.meta[file_name]["headers"] = headers
             self.meta[file_name]["content_type"] = content_type
 
-            if content_type == "text/html":
+            if not content_type:
```

Comment:
I think writing this block in the following way would be nice

```python
if content_type:
	if "text/html" in content_type:
		.....
	elif/else ANOTHER_CONDITION_HERE:
		.....
```

Other than this everything looks good :)